PsfLauncher/main.cpp

```diff
@@ -9,16 +9,20 @@
 #include <sstream>
 
 #include <windows.h>
-#include <psf_constants.h>
-#include <psf_runtime.h>
 #include <shellapi.h>
 #include <combaseapi.h>
 #include <ppltasks.h>
 #include <ShObjIdl.h>
-#include <wil\resource.h>
-#include <wil\result.h>
-#include <TraceLoggingProvider.h>
+#include "StartProcessHelper.h"
 #include "Telemetry.h"
+#include "PsfPowershellScriptRunner.h"
+#include <TraceLoggingProvider.h>
+#include <psf_constants.h>
+#include <psf_runtime.h>
+#include <future>

```

Comment:
No longer needed include

Reply:
Removed.

Reply:
Fixed

---
In reply to: [316806113](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/79#discussion_r316806113) [](ancestors = 316806113)